airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -1468,8 +1470,10 @@ def read_records(
             created_at = record["created_at"]
             if not start_point or cursor_value > start_point:
                 yield record
+                new_state = self._get_updated_state(new_state or self.state, record)
```

Comment:
was there a reason why we have to perform `new_state or self.state` within every invocation of `_get_updated_state()`? I also don't think this properly adhere's to the `_get_updated_state()` interface which expects a mapping whereas we're passing in `None`.

On line 1463, could we just do: `new_state = self.state`?

Reply:
Fixed